### PR TITLE
UI Tweak

### PR DESCRIPTION
**Pull request by HogJonny-AMZN**
This PR implements the following:
- Added two open source fonts (went through legal approval already)
- Updates fonts on the Basic HUD
- Updates Fonts on the setting screen
- On the Start Menu, moves the quit button further from IP field (players were entering the IP address, then accidentally hitting the quit button.) And update to color of some buttons.
- Quit button red on all screens

image

image

image

image

### CODE REVIEW

**Review by mbalfour-amzn — APPROVED** [review]

**Comment by mbalfour-amzn** [comment]
Just a thought, maybe the order should go IP address, Join, Host, Quit, to help enforce that you need to type in the IP address _before_ pressing Join.

**Review by mcphedar — APPROVED** [review]

**Comment by AMZN-Gene** [comment]
> Just a thought, maybe the order should go IP address, Join, Host, Quit, to help enforce that you need to type in the IP address _before_ pressing Join.

+1

**Comment by lmbr-pip** [comment]
Yep IP should be by the Join button button. Host should be separated.

Also primary action colour in O3DE is blue, so Join should be blue, quit should be red IMHO

**Comment by kberg-amzn** [comment]
> Just a thought, maybe the order should go IP address, Join, Host, Quit, to help enforce that you need to type in the IP address _before_ pressing Join.

+1

File: Fonts/Anton/OFL.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+a copy of the Font Software, to use, study, copy, merge, embed, modify,
+redistribute, and sell modified and unmodified copies of the Font
+Software, subject to the following conditions:
+
+1) Neither the Font Software nor any of its individual components,
+in Original or Modified Versions, may be sold by itself.
+
+2) Original or Modified Versions of the Font Software may be bundled,
+redistributed and/or sold with any software, provided that each copy
+contains the above copyright notice and this license. These can be
```
Comment (lmbr-pip) [inline Fonts/Anton/OFL.txt:58]: nit: Do we need todo any other work to highlight this? Do we need to add to the license sections in the README for example?

**Review by lmbr-pip — APPROVED** [review]

File: Fonts/Anton/OFL.txt (hunk 1 above)
Comment (HogJonny-AMZN) [reply Fonts/Anton/OFL.txt:58]: let me double check that

Comment (HogJonny-AMZN) [reply Fonts/Anton/OFL.txt:58]: I'll update readme with copyright notice for these as well as the root license.txt info

**Comment by rainbj** [comment]
Thanks for pushing up those change @HogJonny-AMZN. Looks great. Let me know if you need any additional assets

**Pull request merged**